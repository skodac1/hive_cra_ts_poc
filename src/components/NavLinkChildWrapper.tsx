import { Flex } from "rebass/styled-components";
import {useMemo} from "react";

const NavLinkChildWrapper = ({ children }: any) => {
    const style = useMemo(() => ({
        lineHeight: '145%',
        borderRadius: '4px',
        zIndex: 1,
        cursor: 'pointer',
    }), [])

    return <Flex
        sx={style}
        className='nav-link-child-wrapper'
        alignItems="center"
        py={2}
        px={4}
        mb={3}
        fontSize={2}
        fontFamily="heading"
    >
        {children}
    </Flex>
}

export default NavLinkChildWrapper