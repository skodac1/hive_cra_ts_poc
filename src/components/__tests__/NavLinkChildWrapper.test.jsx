import * as React from 'react';
import { render } from '@testing-library/react';
import NavLinkChildWrapper from "../NavLinkChildWrapper";

describe('<NavLinkChildWrapper />', () => {
    it('matches snapshot', () => {
        const { baseElement } = render(<NavLinkChildWrapper>children</NavLinkChildWrapper>);
        expect(baseElement).toMatchSnapshot();
    });
})