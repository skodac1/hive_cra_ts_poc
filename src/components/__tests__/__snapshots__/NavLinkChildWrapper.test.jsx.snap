// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavLinkChildWrapper /> matches snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-bczRLJ sc-gsnTZi crhLqw hzPtvw nav-link-child-wrapper"
      font-family="heading"
      font-size="2"
    >
      children
    </div>
  </div>
</body>
`;
